skills_DS/api/views.py

```diff
@@ -18,11 +21,22 @@ class FileUploadView(APIView):
 	parser_classes = (FileUploadParser,)
```

Comment:
Should use `MultiPartParser` instead of `FileUploadParser`.
Quote from [Django documentation](https://www.django-rest-framework.org/api-guide/parsers/#multipartparser):
> The FileUploadParser is for usage with native clients that can upload the file as a raw data request. For web-based uploads, or for native clients with multipart upload support, you should use the MultiPartParser instead.